cloudigrade/account/serializers.py

```diff
@@ -5,61 +5,89 @@
 from django.db import transaction
 from django.utils.translation import gettext as _
 from rest_framework import serializers
+from rest_framework.reverse import reverse
 
+import account
 from account import reports
-from account.models import Account
-from account.util import create_initial_instance_events
+from account.models import AwsAccount, Account
+from account.util import create_initial_aws_instance_events
 from util import aws
 
 logger = logging.getLogger(__name__)
 
 
 class AccountSerializer(serializers.HyperlinkedModelSerializer):
-    """Serialize a customer Account for the API."""
+    """
+    Serialize a customer Account for the API.
+
+    The serialized response includes a special "detail" field that links
+    to the cloud provider specific account.
+    """
 
     class Meta:
         model = Account
-        fields = ('id', 'url', 'account_id', 'account_arn')
-        read_only_fields = ('account_id', )
+        fields = (
```

Comment:
Does this guarantee the order the fields will be serialized in? If so: I like alphabetical, but having the dates together can be beneficial.

Reply:
No, and I actually believe DRF always automatically alphabetizes the fields when outputting as JSON.